src/team.njk

```diff
@@ -5,53 +5,57 @@ layout: layouts/nohero.njk
 <div class="team w-full pb-48 ff-bg-light">
     <div class="container m-auto text-left max-w-4xl">
         <div class="grid grid-cols-1 gap-8 sm:grid-cols-2 md:grid-cols-3">
-            {%- for member in team.members -%}
+            {%- for name in team -%}
+            {% set member = team[name] %}
+
             <div class="team-headshot w-full max-w-xs m-auto">
-                {% set imageSrc = ["./images/team/headshot-", team[member].headshot ] | join %}
-                {% set imageAlt = ["Photo of ", team[member].name ] | join %}
+                {% set imageSrc = ["./images/team/headshot-", member.headshot ] | join %}
+                {% set imageAlt = ["Photo of ", member.name ] | join %}
+		{{ imageSrc | log }}
+		{{ imageAlt | log }}
                 {% image imageSrc, imageAlt, [256] %}
```

Comment:
@Pezmc This line doesn't work for me now anymore, and I don't understand the why. Do you have an idea?

Reply:
I tweaked the loop in https://github.com/flowforge/website/pull/520/commits/3b32b4f8838be87ea3a92a9c1529af0f7312dea7, which seems to have fixed your image rendering